src/client/components/MealCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+const dtFormat = new Intl.DateTimeFormat("default", {
+  day: "2-digit",
+  month: "short",
+  year: "numeric",
+  hour: "numeric",
+  minute: "numeric",
+  hour12: false,
+});
+
+function MealCard({ meal }) {
```

Comment:
Did you know you can destructure it further with `{ meal: { title, description } }`?

Reply:
I know that. Anyway thank you.